nx/Makefile

```diff
@@ -0,0 +1,25 @@
+default_target: all
```

Comment:
Instead of changing the `nx` project, let's create a separate project at the root, a sibling to `nx` and `exla`. We just need to settle on the name.

Reply:
Done, `torchx` created.